MMEX/View/Transaction/TransactionEditView.swift

```diff
@@ -96,11 +108,17 @@ struct TransactionEditView: View {
                 Spacer()
                 
                 // Transaction status picker
-                Picker("Transaction Status", selection: $txn.status) {
-                    ForEach(TransactionStatus.allCases) { status in
-                        Text(status.id).tag(status)
+                Menu(content: {
```

Comment:
Hi @georgeef, Is there any reason to replace the picker with the menu?

Reply:
When the Picker is closed, we show only the short name (single letter), because there is no space for more. When the Picker list is expanded (by clicking/touching the Picker), we need a more informative name; the single-letter code is not understandable by the user if it is not explained. The long name in the open Picker helps as a reminder of the encoding.

Picker can show only the same text. Menu+Picker is more flexible, it can show different text when closed and when epanded.

Reply:
This works well in English language, since the single-letter code is the first letter of the long name. With translations of course the long name must also show the code. E.g., "Reconciled" in the expanded list must be translated to "R: [translation-of-Reconciled]".